buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/dependencies/DependenciesMetadataRulesPlugin.kt

```diff
@@ -222,3 +225,17 @@ fun ComponentMetadataHandler.downgradeXmlApis(module: String) {
         }
     }
 }
+
+
+fun ComponentMetadataHandler.replaceCglibNodepWithCglib(module: String) {
+    withModule(module) {
+        allVariants {
+            withDependencies {
+                filter { it.name == "cglib-nodep" }.forEach {
+                    add("${it.group}:cglib:3.2.6")
```

Comment:
:x: The cglib coordinates should be handled centrally in dependencies.gradle instead of copy-pasting the version number into several scripts.